Fix and unify example docs (#19)
lightbend/cloudflow: examples/sensor-data-scala/README.md

## `sensor-data-scala`

### Problem Definition

A simple pipeline that processes events from a wind turbine farm.

### Required configuration

`valid-logger.log-level`

Log level for `*-logger` streamlets to log to.  Ex) `info`

`valid-logger.msg-prefix` - Log line prefix for `*-logger` streamlets to include.  Ex) `VALID`

### Generating data

This example has two ingresses that are combined using a merge operation. Data can be sent to either of the ingresses or to both.

- Pick a test data file from `./test-data`, for example `test-data/04-moderate-breeze.json`
- Send the file to the HTTP ingress using `curl` (see deployment example later on).

To send data to the file ingress, use the following shell script found in the project root directory:

    ./load-data-into-pvc.sh

> The shell script will load a number of files from the `test-data` directory and the ingress will continuously read those files and emit their content to the merge streamlet.

Reviewer: ```suggestion
The shell script will load several files from the `test-data` directory, and the ingress will continuously read those files and emit their content to the merge streamlet.
```

Author: This pre-existed will fix it.
